subprojects/packagefiles/libcurl/add_websockets.patch

```diff
@@ -0,0 +1,57 @@
+Only in curl-8.5.0-ws/: builddir
+Common subdirectories: curl-8.5.0/CMake and curl-8.5.0-ws/CMake
+Common subdirectories: curl-8.5.0/docs and curl-8.5.0-ws/docs
+Common subdirectories: curl-8.5.0/include and curl-8.5.0-ws/include
+Common subdirectories: curl-8.5.0/lib and curl-8.5.0-ws/lib
+Common subdirectories: curl-8.5.0/m4 and curl-8.5.0-ws/m4
+Common subdirectories: curl-8.5.0/meson and curl-8.5.0-ws/meson
```

Comment:
I suppose we don't need this in a patch

Reply:
yes, the diff does need to have the directory, so i just copy the directory, then make the changes and compare directories. So the above gets added. Anyway, is removed now.